src/stream_verifier.rs

```diff
@@ -0,0 +1,58 @@
+use curve25519_dalek::{edwards::EdwardsPoint, scalar::Scalar};
```

Comment:
Perhaps this module could be named `src/verifying/stream.rs` or `src/verifying/streaming.rs`?